datanode/sqlstore/transfers.go

```diff
@@ -217,3 +217,37 @@ func (t *Transfers) getTransfers(ctx context.Context, pagination entities.Cursor
 
 	return transfers, pageInfo, nil
 }
+
+func (t *Transfers) GetAllRewards(ctx context.Context, pagination entities.CursorPagination) ([]entities.Transfer, entities.PageInfo, error) {
```

Comment:
Can you add some unittests please?

Reply:
Added - thought I opened this as a draft for that reason